### ISSUE TITLE
Improvements to documentation on slack availability to new users/developers

### ISSUE THREAD

**Issue by astrochun** [astropy/astropy-project#171]
I know there's been some discussions about Slack, so if this overlaps or has been addressed/discussed please feel free to close it.

**Motivation**
Some OSS projects specify their method of communication (e.g., Gitter, Slack) directly on their README.md with a badge. While Slack is mentioned, it can often be buried. I think there's only one place it's mentioned and that's in the Getting Help page of the `astropy` docs. I just did a search for "slack" in `astropy/astropy` and no hits in any code. By having it front and center, it could help connect new contributors to developers and maintainers faster.

One other advantage that I see is to have specific Slack channels specified for each project (e.g., `astroquery`). This could help facilitate what seems to be a common occurrence that users ask questions in the `general` channel only to be directed to a specific channel.

**Comment by hamogu** [comment]
In my view, slack is more for communication between developers, not a "helpdesk", so maybe it's a good thing to keep it somewhat buried so that the casual looker finds other communication methods first?

**Comment by astrochun** [comment]
Fair point as I understand that there's a burden with maintaining a helpdesk. My thoughts are:
1. Folks who read the README.md are typically developers or new developers. The ability to click on a badge and launch slack in the browser might make it easier
2. While it's primarily for developers, there is a fair amount of Q&A by users
3. It may help reduce issue management. That is partly the reason for spawning GitHub Discussions - to help allow for a community to talk.

**Comment by hamogu** [comment]
I was just giving my personal opinion. I know there's been discussion about discourse and other channels, and personally, I'm not very active on slack, so I might be the wrong person to comment.

**Comment by astrochun** [comment]
@hamogu: I'm definitely glad that you shared your opinions. This was merely a suggestion.

**Comment by astrojuanlu** [comment]
The consensus of gh-152 seemed to be that an OpenAstronomy Discourse would be the best platform for a proper help desk, and if I understand correctly, it will be available soon.

**Comment by bsipocz** [comment]
> proper help desk

the suggestion was never about it being a proper helpdesk (we don't have dedicated resources, operating a proper helpdesk is not a trivial task), but that it will be a user and community forum while keeping slack a developer focused platform as it used to be. (your point no2 is more a bug than a feature, it was never meant to be for user Q&A)

**Comment by astrochun** [comment]
I understand that slack wasn't meant to be for Q&As. My original motivation was about it connecting new contributors to the dev team. Perhaps that's done naturally without bringing slack front and center.

As someone how checks in on the Python for Astronomers FB group, I still see that there's a need for it in an informal manner outside of any formal `astropy` commitments.

There are certainly disadvantages and advantages with each communication methods.

Regardless, if discourse is the way to go, that's great. I still think a badge would be beneficial to point users to the primary method to ask for help.

**Comment by bsipocz** [comment]
The idea to have a badge pointing to discurse is a 👍 from me.

**Comment by hamogu** [comment]
I like that, too. Thanks for bringing it up. I suggest we leave the issue open until we do that to remind us!

**Comment by astrochun** [comment]
I'll wait until the discourse is available and it's properly organized to `astropy` needs. If there's a desire to have custom badge pointing to specific discussions for each repo, I think a full issue creation, fork, and PR creation workflow is possible with the `gh` CLI :smile:

**Comment by pllim** [comment]
We have Discourse now but it is unclear to me revisiting this issue now what was the proposed solution. Is still issue still relevant?
